openghg/store/_flux_timeseries.py

```diff
@@ -184,6 +184,8 @@ def read_file(
         if optional_metadata is None:
```

Comment:
I'm not sure this is necessary, since `check_info_keys` allows `optional_metadata` to be `None`, and we check if `optional_metadata` is `None` on line 189. Probably a similar comment applies to the other data classes.

Reply:
Makes sense to remove - I think this was left over from when additional optional keys were being added to `optional_metadata` so should only impact `Flux` and `FluxTimeseries`.